lib/webdrivers/chromedriver.rb

```diff
@@ -16,7 +16,13 @@ def current_version
       end
 
       def latest_version
-        raise StandardError, 'Can not reach site' unless site_available?
+        unless site_available?
+          cur_ver = current_version
+          raise StandardError, update_failed_msg if cur_ver.nil? # Site is down and no existing binary
+
+          Webdrivers.logger.warn "Can not reach update site. Using existing #{file_name} #{cur_ver}"
+          return cur_ver
```

Comment:
This might not be the right logic here... We might want to throw an error instead of a warning if the driver & browser versions are technically incompatible.

Reply:
Yeah, makes sense. I'm going to go ahead and refactor more code than I'd originally anticipated. Will make a list of all cases and then check them off one by one.